strongbox-web-core/pom.xml

```diff
@@ -395,6 +417,13 @@
             <version>${project.version}</version>
             <type>zip</type>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
```

Comment:
I think if we use plugin to extract DB then this dependency can be moved into plugin dependencies.

Reply:
yes, thanks